core/modules/node/node.module

```diff
@@ -2086,7 +2086,7 @@ function node_block_info() {
   );
   $blocks['content'] = array(
     'info' => t('Existing content'),
-    'description' => t('Embed existing node content as a block onto other pages'),
+    'description' => t('Embed content from existing content items as a block onto other pages.'),
```

Comment:
I'm not a fan of `content from existing content items` Can we move this text change into a follow-up? It's not really relevant here and I don't want to hold this issue up :)

Reply:
Sure thing 👍🏼 ...I've filed https://github.com/backdrop/backdrop-issues/issues/6539 to discuss wording and do that. I'll revert this change.

Reply:
...reverted with [`abebca9` (#4746)](https://github.com/backdrop/backdrop/pull/4746/commits/abebca91e476ee0edf8861ff45be5b633681faa3)